testsuite/WIKI.md

```diff
@@ -0,0 +1,198 @@
+The unit tests and the test data are bundled together in the package **MDAnalyisTests-_release_**. In order to run the tests, this package must be installed in addition to MDAnalysis.
+
+The tests also rely on the `pytest` and `numpy` packages, and require both to run.
+
+# Users
+Install [MDAnalysisTests](MDAnalysisTests) via
+```
+pip install --upgrade MDAnalysisTests
+```
+or download the tar file, unpack, and run `python setup.py install`.
+
+Run the tests by invoking
+```
+python -c 'from MDAnalysis.tests import test; test()'
+```
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error [ask on the user mailing list](http://users.mdanalysis.org) or [raise an issue](/MDAnalysis/mdanalysis/issues).
+
+# Developers #
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error **fix your code** (or *raise an issue*).
+
```

Comment:
This sentence can be deleted. It is enforced now by github that there are no direct pushes to develop